### ISSUE TITLE
export train and validation loss of each individual output of deep learning model with multiple outputs

### ISSUE THREAD

**Issue by yfengli** [rstudio/keras#652]
I am using keras in Rstudio to build a multi-response model, and need to export the train and validation loss value for each output. But it seemingly is hard coded to export only the mean loss. Can we have an option of how to export the loss values, either individual or overall of all outputs?

Or, on the other hand, I have been trying to revise the code myself for the loss values I need. But the issue is after I source the code and run the model building as below,  

(1)  model <- keras_model_sequential() %>% 
      layer_flatten(input_shape = c(3, 5) ) 

I got error 
**Error in keras$models$Sequential(layers = layers, name = name) : 
  attempt to apply non-function**

When add keras:: like below, I can run through 1-2
(1)  model <- keras::keras_model_sequential() %>% 
      keras::layer_flatten(input_shape = c(5, 15) ) 

(2)  model <- model %>% keras::layer_dense(units = 5, activation = 'tanh')
  
then it fail in (3) (either with or without keras::)
(3)    model %>% compile(
   optimizer = optimizer_rmsprop(),
    loss = "mae" 
  )
with error 
**Error in regmatches(ver, regexec("^([0-9\\.]+).*$", ver))[[1]] : 
  subscript out of bounds** 

Any suggestion to solve the problem is appreciated. Thank you!

**Comment by skeydan** [comment]
I think you could try writing a custom callback, in the line of what's shown here:

https://tensorflow.rstudio.com/keras/articles/training_callbacks.html

**Comment by yfengli** [comment]
Thank you, sigrid. I will try it. YF
